common/djangoapps/student/views.py

```diff
@@ -874,20 +849,6 @@ def change_enrollment(request, check_access=True):
 
         # Otherwise, there is only one mode available (the default)
         return HttpResponse()
-
-    elif action == "add_to_cart":
```

Comment:
Just closing the loop on the added discussion on this PR, post merge. The issue is that there was a change to the JavaScript on the registration page. Since the JavaScript is in the .html template file (rather than a separate .js file), the JS changes have to be also applied to our customers customized registration (it is customized since they have additional signup fields to capture). This has been applied and verified on staging. 

As follow-up work, it'd be nice to get the JavaScript out of the the registration.html template and into it's own file at some point.